about/index.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: About the Club
+---
+
+# About
+
+The University of Waterloo Amateur Radio Club is a club at the
+[University of Waterloo](https://uwaterloo.ca) for exploring
+the amateur radio hobby. The club is open to all students of the University, for both
+those who are licensed and those who are not.
+
+The club callsign is VE3UOW.
+
+# Meeting
+
+The club meets on most Friday's at 6:00PM in E2 4402. Anyone is welcome
```

Comment:
I know that vim by [default syntax highlights .md files as Modula 2](http://stackoverflow.com/questions/23279292/how-to-make-vim-understand-that-md-files-contain-markdown-code-and-not-modula), so that may also be a consideration?